packages/sdk/utils/messaging.ts

```diff
@@ -34,6 +34,7 @@ export const sendRequest = <T extends UnknownObject>(
 		requestHashmap[payload.requestId] = {
 			timestamp: new Date(),
 			timeout,
+			// @ts-expect-error can not predefine type in resolve
 			resolve,
```

Comment:
dont use // @ts-expect-error can not predefine type in resolve
use this instead: `resolve: resolve as never`